magento/adobe-stock-integration
# Curated gallery filter is applied when open adobe stock first time

### Description (*)

1.     Install Magento with Adobe Stock Integration
2.     Configure the integration in Stores -> Configuration -> System -> Adobe Stock Integration fieldset

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#603: Curated gallery filter is applied when open adobe stock first time


### Manual testing scenarios (*)
1. Open Adobe stock first time without filter verify images in grid -> https://stock.adobe.com/collections/Kqy5H45lnKzkC5I8JGFXYVwr2ltNwMe7
2. Chose other filters Apply this filters
3. Verify result  there is no gallery_id filter
4. Open adobe stock verify filters applied and gallery_id filter missed
